src/main/java/md/ramaiana/foodmarket/model/Order.java

```diff
@@ -0,0 +1,34 @@
+package md.ramaiana.foodmarket.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+import java.time.OffsetDateTime;
+
+/**
+ * @author Kirill Grosu (grosukirill009@gmail.com), 2/11/2021
+ */
+@AllArgsConstructor
+@Data
+@Builder
+@Table("order")
+public class Order {
+    @Id
+    Integer id;
+    @Column("client_id")
+    Integer clientId;
+    @Column("total_sum")
+    Float totalSum;
+    @Column("created_at")
+    OffsetDateTime createdAt;
+    @Column("deleted_at")
+    OffsetDateTime deletedAt;
+    @Column("processed_at")
+    OffsetDateTime processedAt;
+    @Column("processing_result")
+    String processingResult;
```

Comment:
Order must contain goods, right?
If so, please add property type `List<OrderGood>` properly annotated.
See `@MappedCollection` annotation in `AppUser`.